lib/commands.js

```diff
@@ -19,16 +19,26 @@ function hasAdminPermissions (member) {
 }
 
 function noPermissions (message) {
-  const errorMessage = message.member
-    ? 'You do not have enough permissions to run this command'
-    : 'You need to be in guild channel for this command to work'
+  let errorMessage = ''
+  let fallbackMessage = ''
+  if (message.member) {
+    fallbackMessage = 'You can\'t run this command'
+    errorMessage += fallbackMessage
+    errorMessage += message.channel ? ' inside ' + message.channel : ''
+
+    fallbackMessage += ' here'
+  } else {
+    fallbackMessage = 'You need to be in a guild channel for this command to work'
+    errorMessage += fallbackMessage
+  }
 
-  return message.channel.send({
+  return sendDM(message.author, {
     embed: createEmbed()
       .setTitle('Failed to execute command')
       .setDescription(errorMessage)
+      .addField('Command', message.content)
       .setColor(0xFF0000)
-  })
+  }).catch(err => message.channel.send(fallbackMessage))
```

Comment:
you are not doing anything with err, so either log it like we do everywhere else or just use ()